apps/modernization-ui/src/confirmation/ConfirmationModal.tsx

```diff
@@ -15,6 +15,7 @@ type Props = {
     cancelText?: string;
     onCancel: () => void;
     confirmBtnClassName?: string;
+    confirmBtnDataTestID?: string;
```

Comment:
Question: Is this necessary as a prop? Only 1 confirmation should be displayed on a page at a time. If that is true, you should be able to just set `data-testid='confirmation-btn'` without passing the value in.